packages/next-sass/readme.md

```diff
@@ -133,25 +133,26 @@ module.exports = withSass({
 })
 ```
 
-```js
-// ./pages/_document.js
-import Document, { Head, Main, NextScript } from 'next/document'
-
-export default class MyDocument extends Document {
-  render() {
-    return (
-      <html>
-        <Head>
-          <link rel="stylesheet" href="/_next/static/style.css" />
-        </Head>
-        <body>
-          <Main />
-          <NextScript />
-        </body>
-      </html>
-    )
-  }
-}
+If you are using version less than next@7.0.0
```

Comment:
I think it's best to just remove this from the docs.

Reply:
You say is If you are using version less than next@7.0.0?

Reply:
What I mean is that this whole section can be removed (adding the _document.js), as the docs are always for the latest version.

Reply:
Ok. i'll update this pr